### PR TITLE
move references to nio.env ot nio.conf

### PR DESCRIPTION
**Pull request by tlugger**

### CODE REVIEW

**Comment by tlugger** [comment]
While my commit message may contain a typo, I am somewhat confident that my changes do not contain typos.

File: docs/cli/config.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ Example (project):
 ```bash
 nio cfg project
 ```
-Will prompt you for `PK Host` and `PK Token` values to be put in the `nio.env` file of your project.
+Will prompt you for `PK Host` and `PK Token` values to be put in your projects `nio.conf` file under the [user_defined] section.
```
Comment (jennyknuth) [inline docs/cli/config.md]: project's (possesive)

and do you think `[user_defined]` because copy and paste?

File: docs/cli/new.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The `new` command creates a fresh project from the official [project template](https://github.com/niolabs/project_template) repository. This command will run through the steps of cloning the project, updating all block submodules, and set the initial commit for your project. The new project folder will be made inside the current directory.
 
-You will be prompted to enter values for `PK Host` and `PK Token`. These values will be put in your projects `nio.env` file. Pressing enter without typing in configuration values will leave the values in the `nio.env` unchanged.
+You will be prompted to enter values for `PK Host` and `PK Token`. These values will be put in your projects `nio.conf` file under the [user_defined] section. Pressing enter without typing in configuration values will leave the values in the `nio.conf` unchanged.
```
Comment (jennyknuth) [inline docs/cli/new.md]: project's and `[user_defined]`

File: docs/pubkeeper/README.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ Pubkeeper simplifies security because Pubkeeper clients encrypt the data before
 
 When you create a system in the nio System Designer, by default, a Pubkeeper server is spun up in the cloud. You can override this behavior by selecting something other than “auto” in the “advanced” section of the **create system** dialog. You can see the configuration of the cloud Pubkeeper server if you select a system then click the **edit** icon in the contextual toolbar.
 
-To configure your local project to use the cloud Pubkeeper server, copy the Pubkeeper `host`, `port`, `token`, and `secure` settings shown in your system's edit modal to the following four lines in the <br>`# Pubkeeper Client` section of your project directory's `nio.env` file:
+To configure your local project to use the cloud Pubkeeper server, copy the Pubkeeper `host`, `port`, `token`, and `secure` settings shown in your system's edit modal to the following four lines in the <br>`# Pubkeeper Client` section of your project directory's `nio.conf` file under the [user_defined] section:
```
Comment (jennyknuth) [inline docs/pubkeeper/README.md]: "project directory's" is nice here

`[user_defined]`?

File: docs/running-nio/locally.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ NIO [INFO] Component: ServiceManager status changed from: starting to: started
 
 If you see those logs, nio is up and running. Congratulations!
 
->By default, new projects use port 8181. If port 8181 isn’t available, you'll see an error. Change the NIOPORT value in your project's `nio.env` file to fix this error.
+>By default, new projects use port 8181. If port 8181 isn’t available, you'll see an error. Change the NIOPORT value in your project's `nio.conf` file under the [user_defined] section to fix this error.
```
Comment (jennyknuth) [inline docs/running-nio/locally.md]: `[user_defined]`

File: docs/services/service-design-patterns/environment-variables.md [hunk 5]
```diff
@@ -6,9 +6,9 @@ The nio Platform allows you to define and use variables when configuring your in
 
 ## Access tokens and other secrets
 
-Often a block will need some sort of access token or password in its configuration. Rather than store those in the block config directly, where they are visible in plain text, we recommend using an environment variable for that. Add an entry to your environment variable file `nio.env`.
+Often a block will need some sort of access token or password in its configuration. Rather than store those in the block config directly, where they are visible in plain text, we recommend using an environment variable for that. Add an entry to your configuration file `nio.conf` under the [user_defined] section.
```
Comment (jennyknuth) [inline docs/services/service-design-patterns/environment-variables.md]: `[user_defined]`

File: docs/services/service-design-patterns/environment-variables.md [hunk 6]
```diff
@@ -6,9 +6,9 @@ The nio Platform allows you to define and use variables when configuring your in
...
 ```
-MY_SECRET: p@$$w0rd
+MY_SECRET=p@$$w0rd
```
Comment (jennyknuth) [inline docs/services/service-design-patterns/environment-variables.md]: good catch

File: docs/running-nio/locally.md [hunk 7]
```diff
@@ -93,4 +93,11 @@ tail -f /path/to/my_project/logs/main.log
 ```
 nio shutdown -p {NIOPORT}
 ```
-> NIOPORT is set in your project's nio.env file
+> NIOPORT is set in your project's nio.conf file under the [user_defined] section
+
+---
+## nio.env is obsolete
+
+nio Binaries released after January 26, 2018 no longer use the `nio.env` file for user defined variables. Instead, all of these variables are located under the `[user_defined]` section of `nio.conf`.
```
Comment (jennyknuth) [inline docs/running-nio/locally.md]: comma after 2018

**Pull request merged**